interceptor.go

```diff
@@ -50,47 +52,86 @@ func (i *Interceptor) WrapUnary(next connect.UnaryFunc) connect.UnaryFunc {
 			reporter = i.server
 		}
 
+		var code string
 		if reporter != nil {
-			reporter.ReportStarted(callType, callPackage, callMethod, req.Any())
+			var bytes *prom.CounterVec
+			if reporter.isClient {
+				bytes = reporter.bytesSent
+			} else {
+				bytes = reporter.bytesReceived
+			}
+			if bytes != nil {
+				bytes.WithLabelValues(callType, callPackage, callMethod).Add(float64(proto.Size(req.Any().(proto.Message))))
+			}
+			reporter.ReportStarted(callType, callPackage, callMethod)
+			defer func() { reporter.ReportHandled(callType, callPackage, callMethod, code) }()
+			defer func() {
+				reporter.ReportHandledSeconds(callType, callPackage, callMethod, code, time.Since(now).Seconds())
+			}()
 		}
```

Comment:
Merge these defers

```suggestion
			if bytes != nil {
				bytes.WithLabelValues(callType, callPackage, callMethod).Add(float64(proto.Size(req.Any().(proto.Message))))
			}
			reporter.ReportStarted(callType, callPackage, callMethod)
			defer func() {
				reporter.ReportHandled(callType, callPackage, callMethod, code) 
				reporter.ReportHandledSeconds(callType, callPackage, callMethod, code, time.Since(now).Seconds())
			}()
```

<sub>_[Suggestion by Codeium](https://codeium.com/forge)_</sub>

Reply:
Lol whoops thanks